frontend/src/pages/ProjectSettings/ProjectFilters/ProjectFilters.tsx

```diff
@@ -141,9 +146,12 @@ export const ProjectProductFilters: React.FC<{
 	const { projectId } = useProjectId()
 	const navigate = useNavigate()
 	const { currentWorkspace } = useApplicationContext()
+	const [selectedPreset, setSelectedPreset] =
+		React.useState<DateRangePreset>(DEFAULT_PRESET)
+
 	const [dateRange, setDateRange] = React.useState<DateRange>({
-		start: defaultPresets[1].startDate,
-		end: getNow().toDate(),
+		start: presetStartDate(DEFAULT_PRESET),
+		end: moment().toDate(),
```

Comment:
Manage the start within the component instead of url since multiple of these will be rendered on 1 page and should behave independently